### PR TITLE
@feature/add yusuf file

### PR DESCRIPTION
**Pull request by circutrider21**
Added my file

### CODE REVIEW

File: src/components/yusuf.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+console.log("Hello, Yusuf P2!\n");
```
Comment (tekla010) [inline src/components/yusuf.js:1]: I don't like the \n at the end of your log. Please remove it.

Comment (circutrider21) [reply src/components/yusuf.js:1]: My wife went out

Comment (circutrider21) [reply src/components/yusuf.js:1]: I mean wifi

Comment (circutrider21) [reply src/components/yusuf.js:1]: So I am using my phone

Comment (circutrider21) [reply src/components/yusuf.js:1]: Okay, sounds good.

Comment (Novilab) [reply src/components/yusuf.js:1]: Looks good!!

**Review by Novilab — APPROVED** [review]

**Review by SadiaMohamud — APPROVED** [review]
Nice Job.

**Pull request merged**